mina-archive/templates/cronjob.yaml

```diff
@@ -25,9 +25,7 @@ spec:
                 - /bin/bash
                 - -c
                 - '
-                apk add --no-cache python3 py3-pip;
-                pip3 install --upgrade pip;
-                pip3 install --no-cache-dir awscli;
+                apk add --no-cache aws-cli;
```

Comment:
Minor comment:
Since it is a ephemeral pod running, the `--no-cache` may not be required

Reply:
good point. Removed it